packages/cli/package.json

```diff
@@ -45,6 +45,7 @@
     "find-up": "^4.1.0",
     "fs-extra": "^8.1.0",
     "glob": "^7.1.3",
+    "got": "^10.5.7",
```

Comment:
I'll test the other package I mentioned that was more lightweight and hopefully will not cause to fail in CI

Reply:
Migrated to `node-fetch`  and now CI is ✅